### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Setup_Additional_Keyboard: Tooltips present for the cards which are present in the "VS Code extensions" sections are not accessible using keyboard.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#6208]
### GitHub Tags
#WCAG2.1.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #ChromiumEdge; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #Keyboard; #A11yAuto; #TabStops;

### Environment Details
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Microsoft Edge Version 111.0.1661.62 (Official build) (64-bit)

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Tab till "Docs" link in top navigation and press enter.
3. Tab till "Set up" link in left navigation and press enter.
4. Tab till "Additional components" link present under the "Setup" and press enter.
5. Tab till cards which are present under the "VS Code extensions" sections.
6. Verify whether the tooltips present for the cards which are present in the "VS Code extensions" sections are accessible using keyboard or not.

### Actual Result
Tooltips present for the cards which are present in the "VS Code extensions" sections are not accessible using keyboard.

### Expected Result
Tooltips present for the cards which are present in the "VS Code extensions" sections should be accessible using both keyboard and mouse.

### User Impact
Keyboard user can't be able to get the tooltip information if it is not accessible with it.

### Attachment
![Bug_Tooltips](https://user-images.githubusercontent.com/99243079/230550463-8753e08f-6ded-4047-ab6f-0488af63402e.gif)

**Comment by rzhao271** [comment]
Because we don't control the length of the extension descriptions, and because turning the description paragraph into a scrollable element or lengthening the extension cards to fit the entire description don't seem like the best UX approaches, we can try cutting the description paragraphs from the extension cards. With that solution, the user could still navigate to the extension page to view the full description of the extension.

**Comment by rzhao271** [comment]
Hi @Samhitha66, the issue has been resolved. See https://code.visualstudio.com/docs/setup/additional-components

**Comment by Samhitha66** [comment]
@rzhao271 
Regressed the issue in the below environment
URL:https://code.visualstudio.com/docs/setup/additional-components
As per the above comments - We tried to cutting the description paragraphs from the extension cards. With that solution, the user could still navigate to the extension page to view the full description of the extension.
We verified that now description paragraphs are not there and those information are available in the extension page.
Hence closing this bug.
Please find the attachment for the reference.
#Closed; #Regressed:05-04-23;

![Issue is fixed](https://user-images.githubusercontent.com/99243079/236119391-9224c61c-b106-4516-a47c-9edaa3a980e1.gif)
